packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -590,6 +590,19 @@ export const prodParasPolkadot: EndpointOption[] = [
       color: '#EB027D',
       logo: nodesSubgameSVG
     }
+  },
+    {
```

Comment:
```suggestion
  },
  {
```

This should fix the linting issues.